tabular/src/autogluon/tabular/models/knn/knn_rapids_model.py

```diff
@@ -0,0 +1,35 @@
+import logging
+
+from autogluon.core.constants import REGRESSION
+
+from .knn_model import KNNModel
+
+logger = logging.getLogger(__name__)
+
+# FIXME: Benchmarks show that CPU KNN can be trained in ~3 seconds with 0.2 second validation time for CoverType on automlbenchmark (m5.2xlarge)
+#  This is over 100 seconds validation time on CPU with rapids installed, investigate how it was so fast on CPU.
+#  "2021_02_26/autogluon_hpo_auto.openml_s_271.1h8c.aws.20210228T000327/aws.openml_s_271.1h8c.covertype.0.autogluon_hpo_auto/"
+#  Noticed: different input data types, investigate locally with openml dataset version and dtypes.
+# TODO: Given this is so fast, consider doing rapid feature pruning
+class KNNRapidsModel(KNNModel):
+    """
+    RAPIDS KNearestNeighbors model : https://rapids.ai/start.html
+
+    NOTE: This code is experimental, it is recommend to not use this unless you are a developer.
+    This was tested on rapids-0.18 via:
+
+    conda create -n rapids-0.18 -c rapidsai -c nvidia -c conda-forge -c defaults rapids-blazing=0.18 python=3.7 cudatoolkit=10.1 -y
+    conda activate rapids-0.18
+    pip install --pre autogluon.tabular[all]
+    """
+    def _get_model_type(self):
+        from cuml.neighbors import KNeighborsClassifier, KNeighborsRegressor
```

Comment:
should be lazy import?

Reply:
This is a lazy import right?

Reply:
Right butI meant as an explicit try_import with an informative error message as we have for other lazy imports.

Reply:
added